.github/workflows/mdbook.yml

```diff
@@ -32,19 +32,39 @@ jobs:
       MDBOOK_VERSION: 0.4.30
     steps:
       - uses: actions/checkout@v3
-      - name: Install latest mdBook
-        # installing via cargo takes about 3 minutes, via curl should be seconds
-        run: |
-          tag=$(curl 'https://api.github.com/repos/rust-lang/mdbook/releases/latest' | jq -r '.tag_name')
-          url="https://github.com/rust-lang/mdbook/releases/download/${tag}/mdbook-${tag}-x86_64-unknown-linux-gnu.tar.gz"
-          mkdir mdbook
-          curl -sSL $url | tar -xz --directory=./mdbook
-          echo `pwd`/mdbook >> $GITHUB_PATH
+
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: "latest"
+
+      - uses: baptiste0928/cargo-install@v2
+        with:
+          crate: mdbook-admonish
+          version: "1.9.0"
```

Comment:
Mainly for reproducible builds. If `mdbook-admonish` makes a new version `2.0.0` tomorrow, we wouldn't want our documentation to break. 

Though I guess I can make it use semantic versioning instead of making it use the exact version.

Reply:
I think this might one of those concerns that could happen, but wouldn't really difficult to fix if it did happen. I think I'd just go with latest and let things break (especially if we can implement a preview approach as part of the PR process).

Reply:
Hrm, perhaps (I lean towards no, but happy to leave this as you put it for now)